godtools/ViewControllers/Platform/OnboardingViewController.swift

```diff
@@ -14,35 +14,33 @@ class OnboardingViewController: BaseViewController {
     @IBOutlet weak var page2View: UIView!
     @IBOutlet weak var pageControl: UIPageControl!
     
-    @IBOutlet weak var nowButton: TransparentButton!
-    @IBOutlet weak var laterButton: TransparentButton!
-    @IBOutlet weak var okayButton: TransparentButton!
+    @IBOutlet weak var languagesOkayButton: TransparentButton!
+    @IBOutlet weak var toolsOkayButton: TransparentButton!
```

Comment:
Don't know if these were being used in Storyboards also...just wondering if there were any "orphaned" outlets when these got switched?

Reply:
Yeah, I went through the corresponding .xib file and made sure the appropriate references were where they needed to be.